tests/data/normalization.js

```diff
@@ -57,13 +57,6 @@ describe( 'PasteFromOffice - normalization', () => {
 		editorConfig
 	} );
 
-	generateTests( {
```

Comment:
Shouldn't we remove something from `tests/_data/paste-from-google-docs/lists/index.js` too?

Reply:
Oh, I'll check.

Reply:
I've removed the google docs lists tests.

Reply:
Just to be sure – we still have an integration test for GDocs lists here?

Reply:
I've updated the PR with only normalization fixtures removed.

The fixtures/tests for integration were not touched.